lib/Xud.ts

```diff
@@ -50,15 +51,16 @@ class Xud {
       this.db = new DB(this.config.db);
       await this.db.init();
 
-      this.lndClient = new LndClient(this.config.lnd);
-      await this.lndClient.connect();
+      // setup LND clients and connect if configured
+      this.lndBtcClient = new LndClient(this.config.lndbtc);
+      this.lndLtcClient = new LndClient(this.config.lndltc);
 
+      // setup raiden client and connect if configured
       this.raidenClient = new RaidenClient(this.config.raiden);
-      await this.raidenClient.init();
```

Comment:
Restore this `init` statement, we want it to be called from here and not from the constructor so we can properly wait for it.